packages/usa-layout-grid/src/styles/_usa-layout-grid.scss

```diff
@@ -6,22 +6,25 @@
 
 $namespace-grid: ns("grid");
 
+$custom-breakpoints: map-deep-get($system-properties, breakpoints, extended);
+$all-breakpoints: map-collect($system-breakpoints, $custom-breakpoints);
```

Comment:
> [!note]
> Flagging that layout grid and icon list size use [$system-breakpoints in breakpoints.scss](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/tokens/units/breakpoints.scss) and the other utilities build from [`$system-properties.breakpoints` in _properties.scss](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/_properties.scss#L157). Both seem to collect the same values:
>
> ```scss
> /*breakpoints.scss*/
> $system-breakpoints: general.map-collect(
>   map.get(spacing.$system-spacing, "large"),
>   map.get(spacing.$system-spacing, "larger"),
>   map.get(spacing.$system-spacing, "largest")
> );
> ```
> 
> ```scss
> /*_properties.scss*/
>   breakpoints: (
>     standard:
>       map-collect(
>         map.get($system-spacing, "large"),
>         map.get($system-spacing, "larger"),
>         map.get($system-spacing, "largest")
>       ),
>     extended: (),
>   ),
> ```
> 
> To keep things scoped, we have decided to use `$system-breakpoints` here for now, but can explore replacing it in a future follow-on issue. 
> 
> More details about this can be found in this [comment thread](https://github.com/uswds/uswds/pull/6083/files#r1777767472).